### DISCUSSION TITLE
Weekly Notes

### DISCUSSION THREAD

**Discussion by EgorRazumilov** [Aorta-Detectors/Aorta-Detection#4] [Show and tell]
В этой дискуссии будем писать, что сделали (не сделали) за каждую неделю проекта. Очень хочется, чтобы каждый отписывался в эту дискуссию хотя бы по одному сообщению за раз, например, в каждый понедельник.

**Comment by EgorRazumilov** [comment]
Накидал основу для нашего проекта в виде логики для логина и регистрации пользователей.

**Reply by annamitrofanovaa** [reply]
на прошлых неделях я проходила курсы на степике: по питону, по статистике(частично) и по ml(не все темы еще посмотрела). Скачала данные из issue, просмотрела, думаю что с ними делать. Читала про сегментацию изображений,  еще читала про u-net, на этой неделе буду пробовать что-то писать
